docs/samples/llmisvc/opt-125m/llm-inference-service-facebook-opt-125m-cpu.yaml

```diff
@@ -22,3 +22,8 @@ spec:
           requests:
             cpu: '1'
             memory: 8Gi
+        livenessProbe:
+          initialDelaySeconds: 30
+          periodSeconds: 30
+          timeoutSeconds: 30
+          failureThreshold: 5
```

Comment:
@pierDipi my machine was causing crashlooping without this as it was not starting up on time

```
Pod/facebook-opt-125m-single-kserve-8dc9ff5c6-n4qvr                      Container main failed liveness probe, will be restarted
```